test/DssVestTopUp.test.ts

```diff
@@ -156,30 +157,16 @@ describe("DssVestTopUp", function () {
       });
 
       it("should vest accrued tokens", async function () {
-        const unpaid = await dssVest.unpaid(vestId);
         await topUp.refundUpkeep();
         const vested = await token.balanceOf(topUp.address);
 
-        expect(unpaid).to.eq(vested);
+        expect(vested).to.eq(maxDepositAmt);
       });
 
       it("should return excess payment to surplus buffer", async function () {
-        const unpaid = await dssVest.unpaid(vestId);
-        const topUpTx = await topUp.refundUpkeep();
-
-        // get join event data from mock
-        const topUpRc = await topUpTx.wait();
-        const joinEvent = topUpRc.events?.find(
-          (e) => e.address === daiJoinMock.address
-        );
-        const abiCoder = new ethers.utils.AbiCoder();
-        const [vowAddr, joinAmt] = abiCoder.decode(
-          ["address", "uint256"],
-          joinEvent?.data || ""
-        );
-
-        expect(vowAddr).to.eq(fakeVow);
-        expect(joinAmt).to.eq(unpaid.sub(maxDepositAmt));
+        await topUp.refundUpkeep();
+        const surplusBufferBalance = await token.balanceOf(fakeVow);
+        expect(surplusBufferBalance).to.equal(vestedTokens.sub(maxDepositAmt));
```

Comment:
As stated in a previous comment, using `vestedTokens` is assuming the whole amount is accrued and that's why `unpaid` is called.